src/hooks/use-image-fade-in.ts

```diff
@@ -3,7 +3,7 @@ import { useState } from 'react'
 export const useImageFadeIn = () => {
   const [loaded, setLoaded] = useState(false)
   return {
-    style: { opacity: loaded ? 1 : undefined, transition: 'opacity 200ms ease' },
+    style: { opacity: loaded ? undefined : 0 },
     onLoad: () => {
```

Comment:
uff, alto bug. gracias 🙏🏼 

también deberíamos usar `onLoadingComplete`, en vez de `onLoad`, ya que estamos

```suggestion
    onLoadingComplete: () => {
```